sphinx_doc/tutorials/docs/writing_new_nav2planner_plugin.rst

```diff
@@ -0,0 +1,196 @@
+.. _writing_new_nav2planner_plugin:
+
+Writing a new planner plugin
+****************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+
+.. image:: images/Writing_new_nav2planner_plugin/nav2_rrtconnect_gif.gif
+    :width: 700px
+    :align: center
+    :alt: Animated gif with rrtconnect demo
+
+Overview
+========
+
+This tutorial shows how to create you own planner plugin.
+
+Requirements
+============
+
+- ROS2 (binary or build-from-source)
+- Navigation2 (Including dependencies)
+- Gazebo
+- Turtlebot3
+- `OMPL (binary or build-from-source) <https://ompl.kavrakilab.org/installation.html>`_
```

Comment:
Is there not some rosdep key for OMPL?

Reply:
Not sure, but need to check!

Reply:
OMPL has not been release for eloquent. I think we need to build from source as Movit2 guys does it for now.

Reply:
I filed a ticket https://github.com/ros-gbp/ompl-release/issues/16

Ok, not sure what to do about this at this very moment, but this is a smaller thing and we'll figure it out.